tools/installer/Installer/Controls/InstallSettingsStepControl.xaml

```diff
@@ -127,8 +127,8 @@
                         Overwrite all existing files
                         <LineBreak />
                         <Run Style="{StaticResource small}">
-                            Overwrites any files such as level data, music files and existing user settings.
-                            If this option is off, only overwrites Tomb1Main.exe in order to update the game.
+                            Overwrites any files such as level data and music files. If this option is off, only
+                            overwrites Tomb1Main.exe and other essential files in order to update the game.
```

Comment:
I don't think this checkbox does anything given the new changes – the release .zip contains only files from the new 
 extensions whitelist?

Reply:
There are the three PNG files too - `eidospc.png`, `titleh.png` and `titleh_ub.png`. Should we just remove the checkbox altogether?

Reply:
IMO we should remove this box altogether, and the "install source" shouldn't overwrite the files whereas tomb1main files should. What do you think about this?

Reply:
Sounds good, I'll work on updating it.

Reply:
Updates now in place:
* Everything in the the Tomb1Main embedded zip file will always be written to the target.
* Install source will only write files that aren't in the target folder already.
* Downloaded music and UB files will not be written if they already exist.